cloud-consumption/src/main/java/com/sequenceiq/consumption/service/EnvironmentService.java

```diff
@@ -0,0 +1,31 @@
+package com.sequenceiq.consumption.service;
+
+import javax.inject.Inject;
+import javax.ws.rs.WebApplicationException;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+import org.springframework.stereotype.Service;
+
+import com.sequenceiq.cloudbreak.common.exception.WebApplicationExceptionHandler;
+import com.sequenceiq.environment.api.v1.environment.endpoint.EnvironmentEndpoint;
+import com.sequenceiq.environment.api.v1.environment.model.response.DetailedEnvironmentResponse;
+
+@Service
+public class EnvironmentService {
+
+    private static final Logger LOGGER = LoggerFactory.getLogger(EnvironmentService.class);
+
+    @Inject
+    private EnvironmentEndpoint environmentEndpoint;
+
+    @Inject
+    private WebApplicationExceptionHandler webApplicationExceptionHandler;
+
+    public DetailedEnvironmentResponse getByCrn(String environmentCrn) {
+        try {
+            return environmentEndpoint.getByCrn(environmentCrn);
```

Comment:
Perhaps add a log before the invocation with `debug()` (especially since `LOGGER` is currently not used anywhere :smile: ).